config/routes.rb

```diff
@@ -14,6 +14,7 @@
     member do
       put :edit_serial_number
       put :edit_weight
+      put :add_weight
```

Comment:
I am thinking if this is the best approach. This is not RESTful at all... We should think about it all together @gastonabella @accg14, leave it this way for now.